### PR TITLE
Add missing indent to Mafia Extortion mission

### PR DESCRIPTION
**Pull request by Arachi-Lover**
**Fix (Mission)**

Adds an indent that was missing in the conversation.

### CODE REVIEW

File: data/human/human missions.txt [hunk 1]
```diff
@@ -4057,7 +4057,7 @@ mission "Mafia Extortion"
 				`	"I don't need your credits, just forget it."`
 					goto zerodeal
```
Comment (lumbar527) [inline data/human/human missions.txt:4058]:
```suggestion
```
Not sure if this is the right place but I don't think we need this line.

**Review by lumbar527 — APPROVED** [review]

File: data/human/human missions.txt (hunk 1 above)
Comment (OcelotWalrus) [reply data/human/human missions.txt:4058]: Yes, it's needed.

**Review by OcelotWalrus — APPROVED** [review]

File: data/human/human missions.txt (hunk 1 above)
Comment (Hurleveur) [reply data/human/human missions.txt:4058]: Why would it be?

Comment (OcelotWalrus) [reply data/human/human missions.txt:4058]: Because it wouldn't go to the label.

Comment (Hurleveur) [reply data/human/human missions.txt:4058]: I'm confused
Default behavior is to just go down so it seems to me it would?

Comment (bene-dictator) [reply data/human/human missions.txt:4058]: It's not necessary, no; the conversation goes to the next available label. It's not wrong to leave it there, but as a matter of style I usually remove them.

**Review by bene-dictator — APPROVED** [review]

File: data/human/human missions.txt (hunk 1 above)
Comment (OcelotWalrus) [reply data/human/human missions.txt:4058]: Me I'd let it here because it feels more logical for me. It'd be weird to remove it.

Comment (bene-dictator) [reply data/human/human missions.txt:4058]: Unless Arachi wants to go through the entire mission and remove the technically unneccessary indents, it's fine to leave it as it is. No harm in it.

Comment (OcelotWalrus) [reply data/human/human missions.txt:4058]: Yes.

Comment (Saugia) [reply data/human/human missions.txt:4058]: @Arachi-Lover Do you agree with the above?

Comment (Arachi-Lover) [reply data/human/human missions.txt:4058]: I'll remove them in a bit.

**Review by W1zrad — APPROVED** [review]

**Pull request merged**